include/dxc/DXIL/DxilConstants.h

```diff
@@ -1888,7 +1888,9 @@ const uint64_t ShaderFeatureInfo_WriteableMSAATextures = 0x40000000;
 // WaveMMA slots in between two SM 6.6 feature bits.
 const uint64_t ShaderFeatureInfo_WaveMMA = 0x8000000;
 
-const unsigned ShaderFeatureInfoCount = 31;
+const uint64_t ShaderFeatureInfo_SampleCmpGradientOrBias = 0x80000000;
```

Comment:
Need to add both flags at once, see: #6146.

Reply:
Added.